Model/Config/Backend/SecondChance72.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the MIT License
+ * It is available through the world-wide-web at this URL:
+ * https://tldrlegal.com/license/mit-license
+ * If you are unable to obtain it through the world-wide-web, please send an email
+ * to support@buckaroo.nl so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade this module to newer
+ * versions in the future. If you wish to customize this module for your
+ * needs please contact support@buckaroo.nl for more information.
+ *
+ * @copyright Copyright (c) Buckaroo B.V.
+ * @license   https://tldrlegal.com/license/mit-license
+ */
+namespace Buckaroo\Magento2\Model\Config\Backend;
+
+/**
+ * @method mixed getValue
+ */
+class SecondChance72 extends \Magento\Framework\App\Config\Value
+{
+    /**
+     * Test that the value is a integer within 0 and 180 interval
```

Comment:
function checks 0-72... should it be 0-180 or 0-72?

Reply:
0-72 correct, thank you